### PR TITLE
#168131431 Delete comment posted within the comment thread 

### PR DESCRIPTION
**Pull request by drayzii**
#### What does this PR do?
Allows to delete your comment on a Travel requests
#### Description of Task to be completed?
- Implement deleting comments on requests by their owners
#### How should this be manually tested?
- Clone this repo
- Checkout to branch `ft-delete-comment-168131431`
- Run `npm test`
- Run migrations `npm run db-migrate:dev`
- Run the application with `npm run start:dev` and after you successfully log in make this requests `DELETE` request to `api/v1/requests/comments/{comment id}`
- After you make the request you should see a response that looks like the one in the screenshot
- Alternatively, you can test using the Heroku Review App `https://mervels-bn-backend-stagi-pr-34.herokuapp.com`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168131431](https://www.pivotaltracker.com/story/show/168131431)
#### Screenshots (if appropriate)
image

#### Questions:
N/A

### CODE REVIEW

File: src/controllers/requestController.js [hunk 1]
```diff
@@ -58,7 +57,7 @@ class Requests {
   async getPendingApprovals(req, res) {
     try {
       const field = { status: 'Pending' };
-      const data = await RequestService.findByField(field);
+      const data = await requestService.findByField(field);
```
Comment (willywax) [inline src/controllers/requestController.js]: Why was this changed to lower case ?

**Review by willywax — APPROVED** [review]
Looks good

**Review by Dkabiswa — APPROVED** [review]
Good work

**Review by Paulvitalis200 — APPROVED** [review]
